builtin/credential/aws/path_login.go

```diff
@@ -589,12 +589,26 @@ func (b *backend) pathLoginUpdateEc2(ctx context.Context, req *logical.Request,
 		}
 	}
 
+	identityConfigEntry, err := identityConfigEntry(ctx, req.Storage)
+	if err != nil {
+		return nil, err
+	}
+
+	identityAlias := ""
+
+	switch identityConfigEntry.EC2Alias {
+	case identityAliasEC2InstanceID:
```

Comment:
So am I right in thinking that this is the default path if neither have been set?

Reply:
Yes! The function `identityConfigEntry` above will return default values for both `EC2Alias` and `IAMAlias`.